lib/sprinkle/installers/wget.rb

```diff
@@ -0,0 +1,39 @@
+module Sprinkle
+  module Installers
+    # The wget package installer downloads file from web.
+    #
+    # == Example Usage
+    #
+    #   package :magic_beans do
+    #     description "Beans beans they're good for your heart..."
+    #     wget 'http://example.com/magic_beans.sh', '/usr/local/bin/magic_beans'
```

Comment:
I think path should be optional, since wget will default to the filename already without interference, will it not?